<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintingConfigDialog</class>
 <widget class="QDialog" name="PrintingConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrintingConfigDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_main">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>551</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_moduleSN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>模块编号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_moduleName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>模块名称</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_valueOfModuleName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Printer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>511</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>打印机</string>
    </property>
    <widget class="QLabel" name="label_printerName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>打印机</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_paperSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>纸张</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_paperDirection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>方向</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_paperDirection">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="locale">
      <locale language="Chinese" country="China"/>
     </property>
     <property name="currentText">
      <string extracomment="纵向"/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_paperSize">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="locale">
      <locale language="Chinese" country="China"/>
     </property>
     <property name="currentText">
      <string extracomment="A4"/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_printer">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="locale">
      <locale language="Chinese" country="China"/>
     </property>
     <property name="currentText">
      <string extracomment="纵向"/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_margin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>511</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>页边距（厘米）</string>
    </property>
    <widget class="QLabel" name="label_topMargin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>上边距</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_leftMargin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>左边距</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_bottomMargin">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>下边距</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_rightMargin">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>右边距</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_valueOfTopMargin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_valueOfbottomMargin">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_valueOfRightMargin">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>90</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_valueOfLeftMargin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_offset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>偏移量（厘米）</string>
    </property>
    <widget class="QLabel" name="label_offsetOfHorizontal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>横向偏移</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_offsetOfVertical">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>纵向偏移</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_offsetValueOfHorizontal">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_offsetValueOfVertical">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_moduleSN">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="currentText">
     <string extracomment="纵向"/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_OK">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
